tests/integration/conftest.py

```diff
@@ -61,7 +61,7 @@ def teardown():
         migrate('./inspirehep/demosite/data/demo-records.xml.gz', wait_for_results=True)
         es.indices.refresh('records-hep')  # Makes sure that all HEP records were migrated.
 
-        add_citation_counts(request_timeout=20)
+        add_citation_counts(request_timeout=40)
```

Comment:
@mihaibivol It really sucks that I have to increase this again. Any idea on how to decrease it?